storm-core/src/clj/backtype/storm/daemon/nimbus.clj

```diff
@@ -14,15 +14,29 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 (ns backtype.storm.daemon.nimbus
-  (:import [org.apache.thrift.server THsHaServer THsHaServer$Args])
+  (:import [org.apache.thrift.server THsHaServer THsHaServer$Args]
+           [backtype.storm.generated KeyNotFoundException]
+           [backtype.storm.blobstore LocalFsBlobStore])
```

Comment:
Can we split these up to be separate `(:import ` calls per line?  I know we do it in other places in the code but for consistency it would be nice to only do it one per line.